### DISCUSSION TITLE
Multiple control surfaces (Razer Stream in my case)

### DISCUSSION THREAD

**Discussion by JovissVisuals** [CommandPost/CommandPost#3367] [General]
Has anyone had success running a second loupedeck or Razer?  Just picked up a second Razer stream cause I couldn't get multiple banks working and now no luck running a second device.   Anyone running a second unit of the same control surface?

**Comment by latenitefilms** [comment]
You can definitely run multiple at the same time - I connect all the different versions for testing.
